package.json

```diff
@@ -63,9 +64,21 @@
     "permissionSets": [
       {
         "permissionName": "settings.oai-pmh.enabled",
-        "displayName": "Settings (oai-pmh): display list of settings pages",
+        "displayName": "Display list of settings pages - user can view all oai-pmh settings, but \"Save\" button is disabled",
```

Comment:
Now `displayName` [can be localized](https://folio-project.slack.com/archives/CR3FJRSAD/p1586785138022200). Either create a separate story in order to localize all permissions which exist so far (I think it makes sense to do it only for visible ones) or do it as a part of this PR.

Reply:
Fixed